public/setup/db_data.php

```diff
@@ -408,14 +406,14 @@
   ('10','10','Voir les absences','absences/voir.php',NULL),
   ('10','20','Ajouter une absence','/absence',NULL),
   ('10','30','Informations','/absences/info',NULL),
-  ('15','0','Congés','/holiday/index','config=Conges-Enable'),
-  ('15','10','Liste des cong&eacute;s','/holiday/index','config=Conges-Enable'),
-  ('15','15','Liste des r&eacute;cup&eacute;rations','/holiday/index?recup=1','config=Conges-Enable;Conges-Recuperations'),
-  ('15','20','Poser des cong&eacute;s','/holiday/new','config=Conges-Enable'),
-  ('15','24','Poser des r&eacute;cup&eacute;rations','conges/recup_pose.php','config=Conges-Enable;Conges-Recuperations'),
-  ('15','26','R&eacute;cup&eacute;rations','conges/recuperations.php','config=Conges-Enable'),
-  ('15','30','Informations','conges/infos.php','config=Conges-Enable'),
-  ('15','40','Cr&eacute;dits','conges/credits.php','config=Conges-Enable'),
+  ('15','0','Congés','/holiday','config=Conges-Enable'),
+  ('15','10','Liste des congés','/holiday','config=Conges-Enable'),
+  ('15','15','Liste des récupérations','/holiday?recup=1','config=Conges-Enable;Conges-Recuperations'),
+  ('15','20','Poser des congés','/holiday/add','config=Conges-Enable'),
+  ('15','24','Poser des récupérations','conges/recup_pose.php','config=Conges-Enable;Conges-Recuperations'),
+  ('15','26','Récupérations','conges/recuperations.php','config=Conges-Enable'),
+  ('15','30','Informations','/holiday-info','config=Conges-Enable'),
+  ('15','40','Crédits','conges/credits.php','config=Conges-Enable'),
```

Comment:
Why doing all these in this PR ?

Reply:
I had to update the route to display the information page, yet I also had to remove entities I found on my branch.